code/game/machinery/tcomms/nttc.dm

```diff
@@ -40,18 +40,30 @@
 		"Maintenance Technician" = "engradio",
 		"Station Engineer" = "engradio",
 		// Central Command
-		"Emergency Response Team Engineer" = "dsquadradio", // I know this says deathsquad but the class for responseteam is neon green. No.
+		"Custodian" = "dsquadradio", // I know this says deathsquad but the class for responseteam is neon green. No.
+		"Deathsquad Commando" = "dsquadradio",
+		"Emergency Response Team Engineer" = "dsquadradio",
 		"Emergency Response Team Leader" = "dsquadradio",
 		"Emergency Response Team Medic" = "dsquadradio",
 		"Emergency Response Team Member" = "dsquadradio",
 		"Emergency Response Team Officer" = "dsquadradio",
+		"Intel Officer" = "dsquadradio",
+		"Medical Officer" = "dsquadradio",
+		"Nanotrasen Navy Captain" = "dsquadradio",
 		"Nanotrasen Navy Officer" = "dsquadradio",
+		"Nanotrasen Navy Representative" = "dsquadradio",
+		"Research Officer" = "dsquadradio",
 		"Special Operations Officer" = "dsquadradio",
+		"Sol Trader" = "dsquadradio",
 		"Solar Federation General" = "dsquadradio",
+		"Solar Federation Representative" = "dsquadradio",
 		"Solar Federation Specops Lieutenant" = "dsquadradio",
 		"Solar Federation Specops Marine" = "dsquadradio",
 		"Solar Federation Lieutenant" = "dsquadradio",
 		"Solar Federation Marine" = "dsquadradio",
+		"Supreme Commander" = "dsquadradio",
+		"Thunderdome Overseer" = "dsquadradio",
+		"VIP Guest" = "dsquadradio",
```

Comment:
Honestly I would just gut the entire job name matching system if it is doing it based on the returned ID thing now.

Its probably easier to match based on the states like what is defined in the DMI, since the CC roles have their own set icon for this, same with TSF and similar.
![image](https://user-images.githubusercontent.com/25063394/200170822-9b661738-7596-493a-ba09-c3c4d7328fa3.png)

Reply:
It's meant to (continue to) allow impersonation (in the confines of what is legal for the ongoing event).

It's impossible for any computer on station to give an ID the Rank of "VIP Guest", but it is possible for a malicious actor to change their job title to "VIP Guest" and keep the original rank on the card.
If the malicious actor wears the card and speaks, they would have the same amplification and departmental color, becoming indistinguishable as an impersonator should.

Reply:
Same thing for alternative job titles, without this list, let's say I want to impersonate a person who is a "Custodial Technician", this is technically not a rank, the rank is "Janitor", but I still want them to succeed with the deception, even if they don't have the correct rank to go with the title, simply changing the title to "Custodial Technician" would give them the Service departmental color on comms.

Reply:
Oh I see.

Yeah carry on then.